plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -150,9 +150,9 @@ private Stream<DeltaLakeSplit> getSplits(
         Optional<Long> maxScannedFileSizeInBytes = maxScannedFileSize.map(DataSize::toBytes);
 
         Set<String> predicatedColumnNames = Stream.concat(
-                nonPartitionConstraint.getDomains().orElseThrow().keySet().stream(),
-                columnsCoveredByDynamicFilter.stream()
-                        .map(DeltaLakeColumnHandle.class::cast))
+                        nonPartitionConstraint.getDomains().orElseThrow().keySet().stream(),
```

Comment:
this code formatting changes are not relevant for this change - please remove them from the commit

Reply:
done, removed the code formatting